### PR TITLE
Add JobColored component for enterprise control tower

### PR DESCRIPTION
**Pull request by tomav**
It's an alternate view for `JobStatus` that allow a better view of broken builds for enterprise dashboards.
Colors can probably be improved but it works.

JobColored

### CODE REVIEW

File: styl/index.styl [hunk 1]
```diff
@@ -75,3 +75,17 @@
       padding 0 15px
       font-size: 13px
       color darken($main-txt-color, 20)
+
+.widget {
```
Comment (plouc) [inline styl/index.styl]: You should remove `{`, `}`, `:`, `;` to conform to actual style

File: styl/_view.styl [hunk 2]
```diff
@@ -9,4 +9,11 @@
           &--unknown
             color $unknown-color
           &--failure
-            color $failure-color
\ No newline at end of file
+            color $failure-color
+        &__colored_status
+          &--success
+            background-color $success-color;
```
Comment (plouc) [inline styl/_view.styl]: You should remove `;` to conform to actual style

File: styl/index.styl (hunk 1 above)
Comment (plouc) [inline styl/index.styl]: I should add **stylint** to simplify this

**Comment by plouc** [comment]
@tomav, it's a good idea, it's more obvious, but I think it could be an option to pass to the existing **JobStatus** widget, something like `layout: (default|bold)`, with default to `default` in order to avoid code duplication.

**Comment by plouc** [comment]
did you tried it with other themes ?

**Comment by tomav** [comment]
I'll fix styles and try to use a layout option. You're right about duplication.

**Comment by tomav** [comment]
Ok it's done and working. Tested with other theme, works good but few colors could be probably improved.

**Comment by plouc** [comment]
Thanks @tomav !

**Comment by tomav** [comment]
@plouc when this will be available through `npm`?
Thank you.

**Comment by plouc** [comment]
@tomav, available as `1.1.0`

**Comment by tomav** [comment]
Had to `npm update` again to get latest version.
Works fine now.
Thank you.

**Comment by plouc** [comment]
You're welcome :)

**Pull request merged**